docs/10X_STAR_Solo_alignment.md

```diff
@@ -0,0 +1,328 @@
+   - default: "1MM_CR"
+ - `soloFeatures`: genomic features for which the UMI counts per Cell Barcode are collected
+   - Gene: genes: reads match the gene transcript
+   - SJ: splice junctions: reported in SJ.out.tab
+   - GeneFull: full gene (pre-mRNA): full gene (pre-mRNA): count all reads overlapping genes' exons and introns
+   - GeneFull_ExonOverIntron: count all reads overlapping genes' exons and introns: prioritize 100% overlap with exons
+   - GeneFull_Ex50pAS: full gene (pre-RNA): count all reads overlapping genes' exons and introns: prioritize >50% overlap with exons. Do not count reads with 100% exonic overlap in the antisense direction
+   - default: "GeneFull" 
+ - `soloCellFilter`: cell filtering type and parameters
+   - None: do not output filtered cells
+   - TopCells: only report top cells by UMI count, followed by the exact number of cells
+   - CellRanger2.2: simple filtering of CellRanger 2.2. Can be followed by numbers: number of expected cells, robust maximum percentile for UMI count, maximum to minimum ratio for UMI count. The harcoded values are from CellRanger: nExpectedCells=3000; maxPercentile=0.99; maxMinRatio=10
+   - EmptyDrops_CR: EmptyDrops filtering in CellRanger flavor. Please cite the original EmptyDrops paper: A.T.L Lun et al, Genome Biology, 20, 63 (2019): https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1662-y, Can be followed by 10 numeric parameters: nExpectedCells maxPercentile maxMinRatio indMin indMax umiMin umiMinFracMedian candMaxN FDR simN. The harcoded values are from CellRanger: 3000 0.99 10 45000 90000 500 0.01 20000 0.01 10000"
+   - default: "EmptyDrops_CR"
+  outSAMtype: type of SAM/BAM output. None: no SAM/BAM output. Otherwise, first word is output type (BAM or SAM), second is sort type (Unsorted or SortedByCoordinate)
+   - default: "None"
+### seurat qc
+ - `seurat_qc_min_genes`: minimum number of genes per cell
+ - `seurat_qc_max_genes`: maximum number of genes per cell
+ - `seurat_qc_max_mt`: maximum percent mitochondrial reads per cell
+ - `seurat_qc_normalize_method`: normalization method. One of log_norm or sct
+ - `seurat_qc_num_pcs`: number of PCs to calculate
+
+### Outputs
+ - `star_solo_counts_dir`: Tar gzipped counts output from STAR Solo
+ - `star_solo_bam`: If flag given, aligned reads file
+ - `star_solo_matrix_filtered`: Filtered counts matrix in Cell Ranger-style h5 format
+ - `star_solo_matrix_raw`: Raw counts matrix in Cell Ranger-style h5 format
+ - `star_solo_log_final_out`: STAR align summary stats
+ - `star_solo_junctions`: STAR splice junction result file
+ - `star_solo_cr_mimic_counts`: Tar ball of Cell Ranger-style counts dir from STAR Solo
+ - `seurat_qc_html`: QC HTML Summary
+ - `seurat_qc_rds`: QC rds. See docs for detailed contents of object
+## QC RDS Output
+Given that this is an rds file, for convenience we have posted the results of the R `str()` command on the output
+
+```
+Formal class 'Seurat' [package "SeuratObject"] with 13 slots
```

Comment:
I like having a formal description of the R object but it should be a separate "in-detail" doc that you have. Kind of like I did for the interval stuff in the somatic workflow. You can have a brief paragraph here describing it, then a hyperlink to the highly detailed doc.